packages/ui/src/Tooltip/TooltipAlert.tsx

```diff
@@ -1,31 +1,24 @@
+import WarningOutlinedIcon from '@mui/icons-material/WarningOutlined'
+
 import type { AlertType } from 'ui/src/AlertBox/types'
 import type { TooltipProps } from 'ui/src/Tooltip/types'
 
 import * as React from 'react'
-import Icon from 'ui/src/Icon'
 import IconTooltip from 'ui/src/Tooltip/TooltipIcon'
 
 const TooltipAlert = ({
   alertType,
+  isDeprecated,
```

Comment:
is this set anywhere?

Reply:
oh I see it was set already [here](https://github.com/curvefi/curve-frontend/blob/9209fb035661b3205539c4c7ac293794ee01ae7c/apps/loan/src/hooks/useCollateralAlert.tsx#L13). That comparison with '' also looks fishy

Reply:
Yeah the comparison is for mapping the default of nothing to the existing default of 'info'. The other mapping is from 'danger' to 'error' as there's only three colors we want to use: `info | warning | error`.